src/3-Prayer-Request/PrayerRequestList.tsx

```diff
@@ -0,0 +1,167 @@
+import { useAppDispatch, useAppSelector } from '../TypesAndInterfaces/hooks';
+import { RootState } from '../redux-store';
+import { PrayerRequestListItem, PrayerRequestResponseBody } from '../TypesAndInterfaces/config-sync/api-type-sync/prayer-request-types';
+import theme, { COLORS, FONT_SIZES } from '../theme';
+import PrayerRequestCreate from './PrayerRequestCreate';
+import { StackNavigationProps } from '../TypesAndInterfaces/custom-types';
+import { PrayerRequestTouchable } from './prayer-request-widgets';
+import { ROUTE_NAMES } from '../TypesAndInterfaces/routes';
+
+enum PrayerRequestListViewMode {
+    RECIPIENT = "RECIPIENT",
+    OWNER = "OWNER"
+}
+
+const PrayerRequestList = ({navigation}:StackNavigationProps):JSX.Element => {
+    const dispatch = useAppDispatch();
+
+    const jwt = useAppSelector((state: RootState) => state.account.jwt);
+    const userID = useAppSelector((state: RootState) => state.account.userID);
+    const ownedPrayerRequests = useAppSelector((state: RootState) => state.account.userProfile.prayerRequestList);
+
+    const [prayerRequestsData, setPrayerRequestsData] = useState<PrayerRequestListItem[]>([]);
+    const [viewMode, setViewMode] = useState<PrayerRequestListViewMode>(PrayerRequestListViewMode.RECIPIENT);
+    const [prayerRequestCreateModalVisible, setPrayerRequestCreateModalVisible] = useState(false);
+
+    const RequestAccountHeader = {
+        headers: {
+          "jwt": jwt, 
+        }
+    }
+
+    const renderPrayerRequests = ():JSX.Element[] => 
+        (prayerRequestsData || []).map((prayerRequest:PrayerRequestListItem, index:number) =>
+            <PrayerRequestTouchable
+                key={index}
+                prayerRequestProp={prayerRequest}
+                onPress={() => navigation.navigate(ROUTE_NAMES.PRAYER_REQUEST_NAVIGATOR_ROUTE_NAME, {
+                    params: {PrayerRequestProps: prayerRequest, callback: () => setPrayerRequestsData(prayerRequestsData.filter((prayerRequestListItem:PrayerRequestListItem) => prayerRequestListItem.prayerRequestID !== prayerRequest.prayerRequestID))},
+                    screen: ROUTE_NAMES.PRAYER_REQUEST_DISPLAY_ROUTE_NAME
+                })}
+            />
+        );
+
+    const GET_UserIsRecipientPrayerRequests = async () => {
+        await axios.get(`${DOMAIN}/api/prayer-request/user-list`, RequestAccountHeader).then((response) => {
+            const prayerRequestList:PrayerRequestListItem[] = response.data;
+            setPrayerRequestsData(prayerRequestList);
+        }).catch((error:AxiosError) => console.log(error));
+    }
+
+    const GET_UserIsOwnerPrayerRequests = async () => {
+        await axios.get(`${DOMAIN}/api/user/` + userID + `/prayer-request-list`, RequestAccountHeader).then((response) => {
+            const prayerRequestList:PrayerRequestListItem[] = response.data;
+            
+            //TODO: update personal prayer requests in redux through some loop
```

Comment:
This function is left over from when I initiated our conversation about sourcing owned prayer requests from redux. Because all owned prayer requests get created or deleted locally, this function never gets called. I can delete it.